src/datachain/query/dispatch.py

```diff
@@ -348,7 +330,7 @@ def run_udf_parallel(  # noqa: C901, PLR0912
 
 
 class WorkerCallback(Callback):
-    def __init__(self, queue: multiprocess.Queue):
+    def __init__(self, queue: "multiprocess.queues.Queue"):
```

Comment:
`Queue` is documented as being part of `multiprocess`. AFAICT, `multiprocess.queues` is considered an implementation detail.

Reply:
Hm, yes, you're right. I was doing this to stop typing warnings to pop up, but let me revert this 👍